docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -145,34 +145,51 @@ deployment_groups:
                 - spec: slurm@21-08-8-2
                   prefix: /usr/local
               buildable: False
-      compilers: [gcc@9.3.0 target=x86_64]
+      compilers: [gcc@11.3.0 target=x86_64]
       environments:
       - name: gromacs
         type: file
         content: |
           spack:
             definitions:
-            - compilers:
-              - gcc@9.3.0
-            - mpis:
-              - intel-mpi@2018.4.274
-            - mpi_packages:
-              - gromacs@2020.5
+              - compilers:
```

Comment:
here it seems you have indented the whole thing two spaces to the right, was that intentional?

Reply:
I believe you are right. I think both are valid yaml:

```
# 1 
definitions:
  - compilers:
      - gcc@11.3.0
  - cudas:
      - cuda@11.8.0
  - cuda_mpis:
      - openmpi@4.1.4+cuda
  - mpi_cuda_packages:
      - gromacs@2022.3+cuda+mpi

# 2
definitions:
- compilers:
    - gcc@11.3.0
- cudas:
    - cuda@11.8.0
- cuda_mpis:
    - openmpi@4.1.4+cuda
- mpi_cuda_packages:
    - gromacs@2022.3+cuda+mpi
```
But # 2 matches what we have done elsewhere, so I will update.

Reply:
Done.